admin_manual/maintenance/upgrade.rst

```diff
@@ -41,7 +41,7 @@ Approaching Upgrades
 --------------------
 
 Nextcloud must be upgraded step by step: 
-  * Before you can upgrade to the next major release, Nextcloud upgrades to the latest point release.
+  * Before you can upgrade to the next major release, you need to upgrade to the latest minor version for your current major version.
```

Comment:
```suggestion
  * Before you can upgrade to the next major release, you need to upgrade to the latest point release of your current major version.
```
Nextcloud usually doesn't have minor releases, only patch releases. This wording includes both if there ever is another minor release in the future.